fruitsalad_test.py

```diff
@@ -0,0 +1,86 @@
+import unittest
+import io
+import sys
+from fruitsalad import FruitSaladTool
```

Comment:
@kevinadi Can you add a note to the README with suggested incantation for tests? I used `nosetests -v fruitsalad_test.py`.

Reply:
Done